admin/functions.sh

```diff
@@ -16,7 +16,7 @@ make_temp_dir()
                         DIR => shift(),
                         CLEANUP => 0,
                         TMPDIR => 1,
-                ) or die $!;
+                ) or die $OS_ERROR;
```

Comment:
Did you test this? It might not work without `use English` unless it's being embedded somewhere that already does that?

Reply:
Good catch, just commenting instead in commit https://github.com/metabrainz/musicbrainz-server/pull/3007/commits/1052216aa96379b38c0fac991915a9b3f7b03df1.

Reply:
You forgot to actually change the line :) But I agree with the idea - once you actually add a comment instead, this is ready to merge.

Reply:
Yup, forgot to commit the change; now did in commit https://github.com/metabrainz/musicbrainz-server/pull/3007/commits/49e8b4f64c8bc1b3d53ca7e078baeb371bc3e853